k8s/prod/common/ccd/ccd-case-management-web.yaml

```diff
@@ -30,6 +30,7 @@ spec:
         CCD_DATA_URL: https://gateway.ccd.platform.hmcts.net/data
         CCD_GW_OAUTH2_URL: https://gateway.ccd.platform.hmcts.net/oauth2
         DM_URL: https://gateway.ccd.platform.hmcts.net/documents
+        DM_URL_REMOTE: "^https?://(?:api-gateway\\.preprod\\.dm\\.reform\\.hmcts\\.net|dm-store-prod\\.service\\.core-compute-prod\\.internal(?::\\d+)?)/documents"
```

Comment:
can you fix CODEOWNERS please?

We shouldn't have been requested for review on this

Reply:
it has @hmcts/cdm only